contrib/test_jenkins.sh

```diff
@@ -487,12 +507,24 @@ run_mpi_tests() {
 		test_malloc_hooks_mpi
 		echo "ok 2 - malloc hooks" >> mpi_tests.tap
 
+		run_gtest_release
```

Comment:
why is it added inside run_mpi_tests()? should be in run_gtest()

Reply:
@miked-mellanox suggested to put it close to other release tests to optimize time execution

Reply:
@hoopoepg it would not save that much time given we have multiple workers and make test script harder to maintain.
pls put it in the end of run_gtest(), after valgrind. can use "do_distributed_task" so it would not run on every worker. need to make sure it's not using same worker as coverity.

Reply:
ok, done